xbmc/settings/windows/GUIControlSettings.cpp

```diff
@@ -543,7 +544,7 @@ bool CGUIControlButtonSetting::OnClick()
     }
     else if (controlFormat == "path")
       SetValid(GetPath((CSettingPath *)m_pSetting));
-    else if (controlFormat == "action")
+    else if (controlFormat == "action" || controlFormat == "infoaction")
```

Comment:
Hope to understand you right, you mean is only used as label. Yes and no, is used to open information dialogs on addon (if present). Is a normal button to do somethings, only difference is that the label becomes continuously updated with the cpu usage of the addon dsp mode as string or if not present comes a from KaiToast a information with "Not present from addon...". But is also used on few points only to show information (The first text field on dialog).